libs/documentation/src/lib/components/filters/filters.module.ts

```diff
@@ -15,28 +15,41 @@ import { FiltersHideExpressionModule } from './demos/hideexpression/filters-hide
 import { FiltersHideExpression } from './demos/hideexpression/filters-hideexpression.component';
 import { FiltersShowInactiveFilterValues } from './demos/showInactiveFilterValues/filters-showInactiveFilterValues.component';
 import { FiltersShowInactiveFilterValuesModule } from './demos/showInactiveFilterValues/filters-showInactiveFilterValues.module';
+import { FiltersGroupPanel } from './demos/group-panel/filters-group-panel.component';
+import { FiltersGroupPanelModule } from './demos/group-panel/filters-group-panel.module';
 
 declare var require: any;
 const DEMOS = {
+  groupPanel: {
+    title: 'Filters with Panel Group',
+    type: FiltersGroupPanel,
+    code: require('!!raw-loader!./demos/group-panel/filters-group-panel.component'),
+    markup: require('!!raw-loader!./demos/group-panel/filters-group-panel.component.html'),
+    module: require('!!raw-loader!./demos/group-panel/filters-group-panel.module'),
+    path: 'libs/documentation/src/lib/components/filters/demos/group-panel',
+  },
```

Comment:
This PR - https://github.com/GSA/sam-design-system/pull/565 - updates the logic in retrieving stackblitz generated files, so with those changes, this should also open up on stackblitz